extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -474,8 +475,7 @@ export class AmpA4A extends AMP.BaseElement {
           if (!creativeParts || !creativeParts.signature) {
             return Promise.resolve();
           }
-          this.protectedEmitLifecycleEvent_(
-              'adResponseValidateStart', creativeParts);
+          this.protectedEmitLifecycleEvent_('adResponseValidateStart');
```

Comment:
Yes, you're right.  But this is tricky because there are (potentially) multiple validate start/ends because of multiple services and multiple keys per service.  So you're going to potentially see something like `start, start, end, start, end, start, end, end`.  In practice, that won't happen initially, but it can in the future.

So what's the best approach here?  We could add some unique, per-validation ID such as the key hash to the pings to allow us to reassemble the start/end pairs?  Or don't send a start and only send a validate delta when it completes or errors?